src/styles/_catalog.scss

```diff
@@ -1,22 +1,25 @@
 .catalog {
-  display: flex;
-  flex-wrap: wrap;
-
-  max-width: 944px;
-  margin: 0 auto;
+  display: grid;
+  gap: 48px;
+  justify-content: center;
 
   &__card {
     display: flex;
     justify-content: center;
     align-items: center;
-    width: 200px;
+    width: $card-width;
     height: 300px;
     border: 1px solid #ccc;
     border-radius: 5px;
-    margin-bottom: 48px;
   }
 
-  &__card:not(:nth-child(4n)) {
-    margin-right: 48px;
+  @media (min-width: 488px) {
+    grid-template-columns: repeat(2, $card-width);
+  }
+  @media (min-width: 768px) {
+    grid-template-columns: repeat(3, 200px);
```

Comment:
Replace the fixed width value (200px) with the variable $card-width for consistency